src/browser/provider/built-in/firefox/marionette-client.js

```diff
@@ -8,9 +8,10 @@ import delay from '../../../../utils/delay';
 
 const CONNECTION_READY_TIMEOUT   = 300;
 const MAX_CONNECTION_RETRY_COUNT = 10;
+const MAX_RESIZE_RETRY_COUNT     = 5;
 const HEADER_SEPARATOR           = ':';
 
-const GET_PAGE_SIZE_SCRIPT = 'return {width:document.documentElement.clientWidth, height: document.documentElement.clientHeight}';
```

Comment:
I think the [getWindowDimensionsInfo](https://github.com/DevExpress/testcafe/blob/master/src/browser/provider/built-in/chrome/index.js#L9) function must be shared and used in this place.